modules/radio/react/lib/RadioGroup.tsx

```diff
@@ -43,15 +43,15 @@ const Container = styled('div')<Pick<RadioGroupProps, 'error' | 'grow'>>(
     },
   },
   ({grow}) => grow && {width: '100%'},
-  ({error}) => {
+  ({error, theme}) => {
     let errorBorderColor;
     let errorRingColor;
 
     if (error === ErrorType.Error) {
-      errorRingColor = inputColors.error.border;
+      errorRingColor = theme.palette.error.main;
     } else if (error === ErrorType.Alert) {
-      errorRingColor = inputColors.warning.border;
-      errorBorderColor = colors.cantaloupe600;
+      errorRingColor = theme.palette.alert.main;
+      errorBorderColor = theme.palette.error.darkest;
```

Comment:
Need to do a similar contrast check to [this](https://github.com/Workday/canvas-kit/pull/411/files#diff-0a9f8e72dde3561cb22dfef7dd7accd8R13)